apps/desktop-wallet/src/features/transactionsDisplay/transactionDetailsModal/FTAmounts.tsx

```diff
@@ -11,7 +11,9 @@ const FTAmounts = ({ tx, refAddressHash }: TransactionDetailsModalTxProps) => {
 
   const isMoved = infoType === 'move'
 
-  return <CheckAmountsBox assetAmounts={fungibleTokens} highlight={!isMoved} showPlusMinus={!isMoved} />
+  const assetAmounts = fungibleTokens.map(({ id, amount }) => ({ id, amount }))
```

Comment:
Is this rly necessary? Typescript shouldn't really complain since the props are already there.

Reply:
Removed